README.md

```diff
@@ -36,6 +36,45 @@ Install package via PyPI
 pip3 install dxfeed
 ``` 
 
+## Installation from sources
+
+Reminder: initialize and pull git submodule after cloning the repo:
+```bash
+git submodule init
+git submodule update
+``` 
+
+To install dxfeed from source you need Poetry, Cython and taskipy in addition to the normal dependencies above.
+Poetry provides a custom installer. This is the recommended way of installing poetry according to
+[documentation](https://python-poetry.org/docs/)
+
+For osx / linux / bashonwindows:
+
+```bash
+curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+```
+
+Cython and taskipy can be installed from pypi:
+
+```python
+pip3 install cython taskipy
+``` 
+
+In the project root directory (same one where you found this file after
```

Comment:
If we are talking about poetry and so on, mb lets help our users to do it in "poetry way"? Using .venv and commands like `poetry run task build`. 🤔

Reply:
This is not necessary action. What if users want to install the package globally?